src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -65,51 +65,139 @@ <h4 class="panel-title border-bottom-small" translate>
+                    ng-repeat-start="a in $ctrl.selectableAmounts">
+                <span class="number">&dollar;{{a}}</span>
+              </label>
+              <span ng-repeat-end></span>
+              <label
+                ng-class="{active: $ctrl.customInputActive}"
+                class="btn btn-default-form u-textLeft custom-amount">
+                <div class="form-group form-group-default u-inline">
+                  <input class="form-control form-control-subtle number"
+                        name="amount"
+                        type="text"
+                        step="1"
+                        ng-required="$ctrl.customInputActive"
+                        ng-model="$ctrl.customAmount"
+                        ng-change="$ctrl.changeCustomAmount($ctrl.customAmount)"
+                        ng-focus="$ctrl.customInputActive = true;"
+                        tabindex="-1"
+                        placeholder="{{'Other' | translate}}"
+                        required />
+                </div>
+              </label>
+              <div role="alert" ng-messages="$ctrl.itemConfigForm.amount.$error"
+                  ng-if="($ctrl.itemConfigForm.amount | showErrors)">
+                <div class="help-block" ng-message="pattern" translate>{{'VALID_DOLLAR_AMOUNT_ERROR'}}</div>
+                <div class="help-block" ng-message="required" translate>{{'AMOUNT_EMPTY_ERROR'}}</div>
+                <div class="help-block" ng-message="minimum" translate="AMOUNT_MIN_ERROR" translate-values="{currencyLimit: (1 | currency)}"></div>
+                <div class="help-block" ng-message="maximum" translate="AMOUNT_MAX_ERROR" translate-values="{currencyLimit: (10000000 | currency)}">
                 </div>
               </div>
             </div>
-
+          </fieldset>
+        </div><!-- // panel-body -->
+        <div class="panel-body pt0" ng-if="$ctrl.useV3 !== 'true'">
+          <div ng-if="$ctrl.useSuggestedAmounts">
+            <div class="radio radio-custom-amount"
+                 ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
+              <label>
+                <input name="suggestedAmount"
+                       type="radio"
+                       ng-click="$ctrl.changeAmount(suggested.amount)"
+                       ng-checked="!$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount" />
+                {{$ctrl.suggestedAmount(suggested.amount)}} {{suggested.label}}
+              </label>
+            </div>
+            <div class="radio radio-custom-amount form-inline" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
+              <div class="form-group">
+                <label>
+                  <input 
+                    name="suggestedAmount"
+                    type="radio"
+                    ng-checked="$ctrl.customInputActive"
+                  />
+                  <input
```

Comment:
Was it intentional to lose the `form-control` and `form-control-subtle` classes in this existing version of suggested amounts?

Reply:
Good catch, it had some styles that were a part of the design before v3. I added those classes back in.